### PR TITLE
Forms Automated Test - LPS-131359 | master

### PR DESCRIPTION
**Pull request by leticiachiu**

### CODE REVIEW

**Comment by mlan0** [comment]
Hi @leticiachiu , this PR also looks great to me! 

Could you please again rebase it using the latest master to fix the conflict?

**Comment by leticiachiu** [comment]
done! :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSelectFromListField.testcase [hunk 1]
```diff
@@ -43,14 +43,47 @@ definition {
...
+
+		Form.save();
+
+		FormsAdminNavigator.gotoBack();
+
+		FormsAdminNavigator.gotoEditFormByName(formName = "Form 1");
 
+		AssertElementNotPresent(
+			key_fieldName = "Select from List",
+			locator1 = "FormFields#PARAGRAPH_BODY_TEXT_CONTENT");
```
Comment (mlan0) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSelectFromListField.testcase:86]: @leticiachiu , I think it is a good idea to create a macro for this Assert in the FormFields.macro

There are similar macros for others types of fields, for example: macro viewTextFieldNotPresent

**Comment by mlan0** [comment]
ci:forward

**Pull request closed without merging**